src/EpicsClient/ChannelRequester.h

```diff
@@ -9,16 +9,29 @@ char const *channel_state_name(epics::pvAccess::Channel::ConnectionState x);
 
 class EpicsClientMonitor_impl;
 
+///\Class ChannelRequester
+///\brief provides channel state information for PVs
```

Comment:
I don't think it is worth using \Class \fn etc.
If the brief is one sentence then you don't need to bother with \brief.
The sentence needs a capital letter at the front and a full stop.